databricks/koalas/utils.py

```diff
@@ -358,3 +358,16 @@ def column_index_level(column_index: List[Tuple[str, ...]]) -> int:
         levels = set(0 if idx is None else len(idx) for idx in column_index)
         assert len(levels) == 1, levels
         return list(levels)[0]
+
+
+def tuple_like_strings(items):
```

Comment:
I don't like this naming lol but I think we can change in the separate PR.

Reply:
yeah it's hard to name. any recommended??

Reply:
let's do it later in a separate PR when we handle https://github.com/databricks/koalas/pull/934#discussion_r336932433 comment.

Reply:
okay i will. Thanks for commenting!! 😄